### PR TITLE
FIX: use curl docker img instead of ansible

### LINKED ISSUE #390: Validator API URL needs to be dynamically generated
The URL used for accessing validator APIs is static right now. To enable the user to run multiple validator services on different networks it needs to be dynamic.

See https://github.com/stereum-dev/ethereum-node/blob/main/controls/roles/validator-import-api/tasks/main.yml#L19

### PR DESCRIPTION
**Pull request by stefa2k**
resolves #390 

This PR introduces some key advantages in dealing with the validator APIs of the consensus clients:
- no need to expose validator API port to localhost
- it's now possible to run the same validator client multiple times and import to a specific one (e. g. using them on different networks)

todos:
- [x] clean code
- [x] fix process with nimbus (doesn't work right now)
- [x] clean or preserve curl container? image?
- [x] docker container user
- [x] import multiple accounts at once
- [x] fix delete validator via API

### CODE REVIEW

**Comment by stefa2k** [comment]
also resolves #397

File: controls/roles/validator-import-api/tasks/import.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+- name: Generate POST data
+  set_fact:
+    validator_import_post_data: |
+      {
+        "keystores": {{ item.content | to_json }},
+        "passwords": {{ item.passwords | to_json }}
```
Comment (gbayasgalan) [inline controls/roles/validator-import-api/tasks/import.yaml:7]: maybe missing `with_items: ...`?

File: controls/roles/validator-import-api/tasks/main.yml [hunk 2]
```diff
@@ -1,103 +1,76 @@
...
-      body:
-        keystores: "{{ item.content }}"
-        passwords: ["{{ validator_password | quote }}"]
-      body_format: json
-    become: yes
-    with_items: "{{ validator_keys }}"
-  when: consClient == "prysm"
+  - include_tasks: import.yaml
+    vars:
+      validator_port: 7500
```
Comment (gbayasgalan) [inline controls/roles/validator-import-api/tasks/main.yml:30]: missing `api_token`

Comment (stefa2k) [reply controls/roles/validator-import-api/tasks/main.yml:27]: done

**Comment by stefa2k** [comment]
Can't fix issue with Nimbus right now due to https://github.com/status-im/nimbus-eth2/issues/3760

**Comment by stefa2k** [comment]
> Can't fix issue with Nimbus right now due to [status-im/nimbus-eth2#3760](https://github.com/status-im/nimbus-eth2/issues/3760)

resolved

**Review by NeoPlays — COMMENTED** [review]
lgtm 🚀
@gbayasgalan should also have a look 😅

**Review by gbayasgalan — APPROVED** [review]
lgtm too

**Pull request merged**